docs/DeveloperGuide.md

```diff
@@ -1,34 +1,662 @@
+ 
+ The sequence diagram for searching for an event is as shown below:
+
+ ![](EventDiagram/SequenceDiagram/CommandSearchEvent.png)
+ 
+ 
+**3.4.4. Displaying countdown to upcoming events** `CommandEventCountdown`
+
+Current Implementation
+The `CommandEventCountdown` class in `seedu.duke.event` handles displays the countdown as an additional feature in the eventlist.
+ 
+It implements the following operation:
+*`CommandEventCountdown#execute()` -  displays countdown feature for all upcoming `Event` in the `EventList`.
+
+The sequence diagram for displaying countdown is as shown below:
+
+![](EventDiagram/SequenceDiagram/CommandEventCountdown.png)
+
+**3.4.5. Mark an event as completed** `CommandEventStatus`
+
+Current Implementation
+The `CommandEventStatus` class in `seedu.duke.event` handles marking of an event. It can manually mark an event as done.
+ 
+It implements the following operation:
+*`CommandEventStatus#execute()` -  Marks an `Event` in the `EventList` as done.
+
+The sequence diagram for marking an event as done is as shown below:
+
+![CommandEventStatus](EventDiagram/SequenceDiagram/CommandEventStatus.png)
+
+**3.4.6. Add/delete event participants feature** `CommandEventAdd` , `CommandEventDel` 
+3.4.6.1. Current Implementation  
+The `CommandAddEventAttendance` class in `seedu.duke.event` handles the adding of events. According to the `userInput`, it adds a new event to the `EventList`. 
+The `CommandEventDel` class in the same package handles deleting of a event. It deletes an `Event` instance according to the index provided by `userInput` from the `EventList`.  
+They implement the following operations:  
+* `CommandEventAdd#execute()` - Adds a new event into the `EventList` according to `userInput`.  
+* `CommandEventDel#execute()` - Deletes an event from `EventList` or deletes all the events in the list. 
+To delete a particular event, enter the index of the event.
+To delete all the events in the list, enter `all` instead of the index of the event.
+
+Given below is an example usage scenario and how add/delete event function behaves at each step.  
+
+Step 1. The user launches the application for the first time.   
+
+Step 2. The user executes `event addEvent /n arduino course /d 2020-12-30 /t 8pm` command to add a new event with the name "arduino course", 
+the date of the event "2020-12-30" and the time "8pm" into event list. 
+The `event addEvent` command calls `CommandEventAdd#execute()`, then `EventList` will add a new `Event` with event name as `iphone12`, date as `2020-12030` and time as `8pm`.  
+
+Step 3. The user executes `event delEvent 1` command to delete the 1st event in the event list. The `event delEvent`
+command calls `CommandEventDel#execute()`, causing the `Event` at index 1 to be removed from `EventList`.  
+
+The sequence diagram for adding an event is as shown below:
+
+![CommandAddEventAttendance](EventDiagram/SequenceDiagram/CommandAddEventAttendance.png)
```

Comment:
<img width="749" alt="Screenshot 2020-10-30 at 12 25 43" src="https://user-images.githubusercontent.com/59906930/97659959-47b97b80-1aab-11eb-9b88-06110b95b2de.png">

The image did not load properly for this, maybe the file location is wrong?